src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let start = this[0];
+    let sameShitAsIndex = 1;
```

Comment:
Bad variable name
Rename the variable according to the rules for naming variables
![image](https://user-images.githubusercontent.com/70009205/228290925-d9b71d1f-54db-4ed2-9c66-1fa5eb0562f3.png)
ex `startIndex` `commonIndex` `innerIndex` `index` (and in loop will be let i = index)